UNLV-CS-GANG/gatekeeper
# Added Styling to Event Form

# Changes

- Styling of event form
- Removal of burger icon
- Removal of dropdown 

## How to Test
Try changing the window size and also check on responsiveness by using different device dimensions on the developer tools menu.
